include/sound/sof.h

```diff
@@ -103,5 +103,6 @@ struct sof_dev_desc {
 int sof_nocodec_setup(struct device *dev, const struct snd_sof_dsp_ops *ops,
 		      int (*pcm_dai_link_fixup)(struct snd_soc_pcm_runtime *rtd,
 						struct snd_pcm_hw_params *params));
+int snd_sof_dai_get_ssp_mclk(struct snd_soc_pcm_runtime *rtd);
```

Comment:
Hmm, a minor thing but why we use two prefixes in the same header, "sof_foobar()" and "snd_sof_foobar()"? This doesn't seem right.